### PR TITLE
Update registry 

### PR DESCRIPTION
**Pull request by milosgajdos**
New release: https://github.com/distribution/distribution/releases/tag/v2.8.1

Signed-off-by: Milos Gajdos <milosthegajdos@gmail.com>

### CODE REVIEW

File: library/registry [hunk 1]
```diff
@@ -2,7 +2,7 @@ Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
 
-Tags: 2.8.0, 2.8, 2, latest
+Tags: 2.8.1, 2.8.0, 2.8, 2, latest
```
Comment (thaJeztah) [inline library/registry:5]: 2.8.0 should not be in here

Comment (milosgajdos) [reply library/registry:5]: Right

Comment (milosgajdos) [reply library/registry]: https://github.com/docker-library/official-images/pull/12001/commits/3989b41fa61384db20809af75b9e9d1e46e56ba8

**Review by thaJeztah — COMMENTED** [review]
LGTM

boooh, you blocked external reviews? 😭😭
96E5818D-A0B6-4835-A838-EC1D54402E26

**Comment by thaJeztah** [comment]
Thanks!!

**Comment by thaJeztah** [comment]
OHMAN, that screenshot is HUUUGE when looking at it in Desktop mode 😂 (posted it from my phone)

**Pull request merged**